### PR TITLE
Run hlint in --refactor mode (inplace if possible)

### LINKED ISSUE #69: Run hlint in --refactor mode
Is it possible to add this as the project it is right now? Just like `ormolu` is run with `--mode inplace`? 😉

### PR DESCRIPTION
**Pull request by kutyel**
Fixes #69 

1) Maybe I need to add https://github.com/mpickering/apply-refact somewhere? 
2) Maybe we want to somehow opt-out of the "inplace" mode in the case of hlint 🤔
3) Is it possible to add the label `hacktoberfest-accepted` to this PR? 🙏 

Happy hacktoberfest! 🥳

### CODE REVIEW

File: modules/hooks.nix [hunk 1]
```diff
@@ -53,9 +53,9 @@ in
       hlint =
         {
           name = "hlint";
-          description =
+          description =
```
Comment (domenkozar) [inline modules/hooks.nix:56]:
```suggestion
          description =
```

**Comment by dhess** [comment]
This doesn't work with `hlint-3.2.1`, at least:

```
hlint....................................................................Failed
- hook id: hlint
- exit code: 1

hlint: Refactor flag can only be used with an individual file
CallStack (from HasCallStack):
  errorIO, called at src/HLint.hs:194:14 in hlint-3.2.1-5SE9YOnH4UyLLY7L7KPrCd:HLint
```

**Comment by domenkozar** [comment]
That's strange, I think pre-commit does pass each file as an argument.

**Comment by dhess** [comment]
Yes, this is a fully checked-in & committed source tree.

**Comment by domenkozar** [comment]
I've reverted this as it seems that hlint only supports this option iff one file is passed: https://github.com/ndmitchell/hlint/blob/master/src/HLint.hs#L194

**Pull request merged**